stellar-contract-env/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "stellar-contract-env"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+stellar-contract-env-guest = { path = "../stellar-contract-env-guest", optional = true }
+stellar-contract-env-host = { path = "../stellar-contract-env-host", optional = true }
+
+[features]
+default = ["host"]
```

Comment:
We should leave this default out I think, and force the importer to specify a feature. This is actually better for one-or-the-other features I think, because we don't have to disable the default feature set.